charts/kong/templates/_helpers.tpl

```diff
@@ -245,7 +267,7 @@ The name of the service used for the ingress controller's validation webhook
 {{- range .Values.plugins.secrets -}}
   {{ $myList = append $myList .pluginName -}}
 {{- end }}
-{{- $myList | join "," -}}
+{{- $myList | uniq | join "," -}}
```

Comment:
I don't see why this should be added. Kong will not error out if the same plugin is specified multiple times. There will be other problems that will happen because of the deployment.

Just a comment. Even if keep this around, it is ok.